addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/discovery/FreeboxDiscoveryService.java

```diff
@@ -99,6 +100,9 @@ public void onDataFetched(ThingUID bridge, LanHostsConfig hostsConfig) {
                     if (thingUID != null) {
                         logger.trace("Adding new Freebox Network Device {} to inbox", thingUID);
                         Map<String, Object> properties = new HashMap<>(1);
+                        if (!hostConfig.getVendorName().isEmpty()) {
```

Comment:
The vendor is always set, because you define it in the thing type, don't you?
Otherwise, I would expect that `hostConfig.getVendorName()` can be null, which would result in an NPE. Or am I missing something?